src/server/scripts/EasternKingdoms/SunwellPlateau/boss_eredar_twins.cpp

```diff
@@ -119,7 +119,9 @@ struct boss_sacrolash : public BossAI
 
             scheduler.CancelGroup(GROUP_SPECIAL_ABILITY);
             ScheduleTimedEvent(20s, [&] {
-                Unit* target = SelectTarget(SelectTargetMethod::MaxThreat, 1, 100.0f);
+                std::list<Unit*> targets;
+                SelectTargetList(targets, 5, SelectTargetMethod::MaxThreat, 0, 100.0f, true, false);
```

Comment:
```suggestion
                SelectTargetList(targets, 6, SelectTargetMethod::MaxThreat, 0, 100.0f, true, false);
```

The issue reads to me as 6. @amed80 could you confirm whether this is 5 or 6?
> Grand Warlock Alythess's Conflagration should target randomly one of the top 6 on Lady Sacrolash threat table apart from the current Lady Sacrolash's target.

Reply:
In the examples the lowest in threat that was selected was Top6. So we need to select 5 targets because we exclude the tank